Makefile

```diff
@@ -0,0 +1,37 @@
+# Makefile for Asyn rzbrp100 support
+#
+# Created by dpk62 on Mon Oct 18 10:53:13 2021
+# Based on the Asyn streamSCPI template
+
+TOP = .
+include $(TOP)/configure/CONFIG
+
+DIRS += configure
+DIRS += $(wildcard *Sup)
+DIRS += $(wildcard *App)
+DIRS += $(wildcard *Top)
+DIRS += $(wildcard iocBoot)
+
+# The build order is controlled by these dependency rules:
+
+# All dirs except configure depend on configure
+$(foreach dir, $(filter-out configure, $(DIRS)), \
+    $(eval $(dir)_DEPEND_DIRS += configure))
+
+# Any *App dirs depend on all *Sup dirs
+$(foreach dir, $(filter %App, $(DIRS)), \
+    $(eval $(dir)_DEPEND_DIRS += $(filter %Sup, $(DIRS))))
+
+# Any *Top dirs depend on all *Sup and *App dirs
+$(foreach dir, $(filter %Top, $(DIRS)), \
+    $(eval $(dir)_DEPEND_DIRS += $(filter %Sup %App, $(DIRS))))
+
+# iocBoot depends on all *App dirs
+iocBoot_DEPEND_DIRS += $(filter %App,$(DIRS))
+
+# Add any additional dependency rules here:
+
+include $(TOP)/configure/RULES_TOP
+

+ioctests:

+	.\system_tests\run_tests.bat


```

Comment:
Some nasty characters at end-of-lines here which I think causes build trouble - I think this is fixed in latest version of IOC gen but can you fix here please?

Reply:
Characters are <CR>.  Now replaced with <CR><LF>.

Reply:
Also replaced `<TAB>` with four spaces for consistency.

Reply:
Replaced spaces with `<TAB>` to enable successful build.